src/sdk/model/edge/edge-gateway-l2vpn/edge-gateway-l2vpn-peer-site.ts

```diff
@@ -43,6 +46,9 @@ export class EdgeGatewayL2VpnPeerSite {
    * @returns {Array<number>}
    */
   get vnics(): Array<number> {
+    if (!this._json.vnics) {
```

Comment:
I think all these if return statements are good cases to use the ternary operator, instead of each being 4 lines they'll be one

Reply:
It could be simplified even a little more. return this._json.vnics || []